docs/setting-your-IDE.md

```diff
@@ -0,0 +1,91 @@
+## The basics
+
+If you are interested in being able to quickly debug and develop Mailchimp Open Commerce, consider installing these useful tools to help you.
+
+- You should use [nvm](https://github.com/nvm-sh/nvm) to install and manage NodeJS.
+- Visual Studio Code is a preferred text editor but you can use Sublime Text, Emacs, Vim or any other editor of your choice.
+
+## Browser Extensions
+
+### Chrome
+
+- [React Developer Tools](https://chrome.google.com/webstore/detail/react-developer-tools/fmkadmapgofadopljbjfkapdkoienihi)
+- [Apollo Client Developer Tools](https://chrome.google.com/webstore/detail/apollo-client-developer-t/jdkknkkbebbapilgoeccciglkfbmbnfm)
+- [MobX Developer Tools](https://chrome.google.com/webstore/detail/mobx-developer-tools/pfgnfdagidkfgccljigdamigbcnndkod)
+- [Altair GraphQL Client](https://chrome.google.com/webstore/detail/altair-graphql-client/flnheeellpciglgpaodhkhmapeljopja)
+- [Meteor DevTools](https://chrome.google.com/webstore/detail/meteor-devtools/ippapidnnboiophakmmhkdlchoccbgje)
+- [z-context](https://chrome.google.com/webstore/detail/z-context/jigamimbjojkdgnlldajknogfgncplbh)
+
+### Firefox
+
+- [Altair GraphQL Client](https://addons.mozilla.org/en-US/firefox/addon/altair-graphql-client/)
+- [React Developer Tools](https://addons.mozilla.org/en-US/firefox/addon/react-devtools/)
+
+## Code Editor Extensions
+
+### [Visual Studio Code](https://code.visualstudio.com/)
```

Comment:
Let's limit these to ones that are more MOC specific and add some info on why you should add them. I think just a list of plugins is not going to be that helpful.